### PR TITLE
Fix "users/status/ids" API documentation

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

File: v4/source/status.yaml [hunk 1]
```diff
@@ -60,14 +60,22 @@
...
-    get:
+    post:
       tags:
         - status
       summary: Get user statuses by id
       description: |
         Get a list of user statuses by id from the server.
         ##### Permissions
         Must be authenticated.
+      parameters:
```
Comment (cpanato) [inline v4/source/status.yaml]: Hello @thePanz thanks for this update
just a small nit

can you change the following parameters to fix the build issue and also the fix the definition

 - `name` should be `post`
 - in the `schema` the items should be:
  ```
  items:
    type: string
  ```

the whole `parameters` section:
```
parameters:
    - name: post
       in: body
       description: List of user ids to fetch
       required: true
       schema:
           type: array
           items:
             type: string
```

Comment (thePanz) [reply v4/source/status.yaml]: Updated the MR with your comments :)

**Comment by cpanato** [comment]
cool! many thanks, just a very small request, can you please squash the commits into one? thanks so much again.

**Comment by thePanz** [comment]
@cpanato I'll work on that during the next days. 
btw: can you check if github has a "merge and squash" option for PRs? it would do it too :)

**Comment by cpanato** [comment]
thats true 😄  forgot that :)

**Comment by cpanato** [comment]
thanks again!

**Pull request merged**